SU2_PY/SU2/run/deform.py

```diff
@@ -85,7 +85,9 @@ def deform ( config, dv_new=None, dv_old=None ):
         return info
     
     # setup mesh name
-    suffix = 'deformed'
+    suffix = 'deform'
+    if konfig['ENABLE_REMESHING'] == 'YES':
+        suffix += 'ed'
```

Comment:
@danielmayer please change that to your needs if this here doesn't fit, but the default suffix should not change for other regression cases